locust/static/style.css

```diff
@@ -234,6 +234,24 @@ a {
     white-space: pre;
 }
 
+#charts {
+    width: 100%;
+}
+#charts .chart {
+    height: 350px;
+    margin-bottom: 30px;
+    box-sizing: border-box;
+    border: 1px solid #1b1f1f;
+}
+#charts p.note {
```

Comment:
Minor nitpick but generally tag selectors are discouraged unless style is specifically tied to tag. In this case, the layout could change and lose the style.